packages/examples/README.md

```diff
@@ -0,0 +1,38 @@
+# node-wot Examples
+
+## Demo Serviants
+
+Demo serviants examples are located in `src\demo-servients`.
+
+## Script Examples 
+
+Script examples are located in
+
+* `src\scripts`
+* `src\proxy-scripts`
+* `src\testthing`
+
+The idea of these folders is to use TypeScript to work on the examples which offers support in being up-to-date with the current API.
+
+see https://github.com/eclipse/thingweb.node-wot/issues/171.
+
+### Workflow
+
+1. Run `npm run build`
+2. Remove the following 3/4 lines in JS files of folder `dist/` 
```

Comment:
I'm not sure what I have to do here? 3/4 lines (do you mean 3-4 lines)?

Reply:
the `tsc` command creates the 3/4 lines mentioned below which at the moment need to be removed manually. In the future we should have an automates travis script